File: Submissions/T19/M1/m1.c
#include <stdio.h>
int main() {
    int nilai[5];
    printf("Masukkan 5 nilai : ");
    for (int i = 0; i < 5; i++) scanf("%d", &nilai[i]);
    double rata = 0;
    for (int i = 0; i < 5; i++) rata += nilai[i];
    rata /= 5;
    char grade;
    if (rata >= 80) grade = 'A';
    else if (rata >= 70) grade = 'B';
    else if (rata >= 60) grade = 'C';
    else if (rata >= 50) grade = 'D';
    else grade = 'E';
    printf("Rata-rata : %.2f\n", rata);
    printf("Grade : %c\n", grade);
}

File: Submissions/T19/M1/m1.py
nilai = list(map(int, input("Masukkan 5 nilai : ").split()))
rata = sum(nilai) / 5
grade = 'A' if rata >= 80 else 'B' if rata >= 70 else 'C' if rata >= 60 else 'D' if rata >= 50 else 'E'
print(f"Rata-rata : {rata:.2f}")
print(f"Grade : {grade}")
